packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1054,6 +1063,8 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.call?.on('stateChanged', this.subscribeToCaptionEvents.bind(this));
     /* @conditional-compile-remove(call-transfer) */
     this.call?.feature(Features.Transfer).on('transferRequested', this.transferRequested.bind(this));
+    /* @conditional-compile-remove(capabilities) */
+    this.call?.feature(Features.Capabilities).on('capabilitiesChanged', this.capabilitiesChanged.bind(this));
```

Comment:
I am a little cautious about chains with ? after what happened on Friday when we tried to get raised hand. Can we confirm that if we can resolve the call then we can always resolve the capabilities feature and if so we can guarantee that it will never be undefined?

Reply:
What happened on Friday was a renaming of properties from Calling SDK. For example, property `ParticipantCapabilities.muteUnmuteMic` was renamed `ParticipantCapabilities.unmuteMic` in one of their beta releases. Are we supposed to prepare for any possible renaming Callling SDK will do? How can I guarantee that a property in the API is then for one version will be renamed in the next version of Calling SDK?